README.md

```diff
@@ -69,84 +131,50 @@ furyagent
 │   ├── openvpn-client
 │   └── ssh-keys
 ├── backup
-│   ├── etcd
-│   └── master
+│   └── etcd
 └── restore
-    ├── etcd
-    └── master
+    └── etcd
 ```
 
-## Workflow
+## Configure Furyagent
 
-1. Write a [`furyagent.yml`](furyagent.yml)
-2. Generate certificates (by hand at the moment)
-3. `furyagent init -d /path/to/cert/dir --config /path/to/furyagent.yml [etcd|master]` to upload certificates
-4. Then on the nodes: `furyagent configure --config /path/to/furyagent.yml [etcd|master]` to download the certificates to the correct directory specified in the config file
-5. if needed: to backup the state of etcd through `furyagent backup --config /path/to/furyagent.yml etcd`
-6. if needed: to restore the state of etcd, stop etcd, run `furyagent restore --config /path/to/furyagent.yml etcd`, restart etcd
+`furyagent` uses an utility S3 Bucket. To configure access, `furyagent` reads a `Furyagent.yml` config file.
 
-## Contributing
+```bash
+storage:
+  provider: s3  
+  url: "http://s3.eu-central-1.amazonaws.com"
+  aws_access_key: <AWS_ACCESS_KEY>
+  aws_secret_key: <AWS_SECRET_KEY>
+  bucketName: <AWS_BUCKET_NAME>
+  region: <AWS_BUCKET_REGION>
+```
 
-We still use `go mod` as golang package manager. Once you have that installed you can run `go mod vendor` and `go build` or `go install` should run without problems
+The `storage` section contains the AWS credentials and the location of the S3 bucket. Make sure that the AWS service account has read/write permissions to the bucket.
```

Comment:
"... and to store this file encrypted or in a secure way" ?

Reply:
✅